brambling/templates/brambling/user/billing.html

```diff
@@ -26,116 +26,93 @@
 	{{ block.super }}
 	<h2 class="sr-only">Billing settings</h2>
 
-	{% include "brambling/forms/_top_error_list.html" %}
+	{# {% if stripe_live_settings_valid or stripe_test_settings_valid %} #}
```

Comment:
Originally this was not commented out; basically, it was set up to only display the stripe / dwolla settings if they were actually configured. I had commented it out for testing and forgot to re-comment it.

But looking at it again, I think we should just always display the stripe section; we already have messaging inside this section if the relevant stripe configuration isn't set up, and otherwise this screen is blank and confusing. So I'll remove the comment and de-indent the code inside it.